### PR TITLE
New imagery for the Canton Zürich

### PR DESCRIPTION
**Pull request by simonpoole**
Update from https://github.com/simonpoole/osm-layer-index

### CODE REVIEW

File: sources/europe/KantonZuerich10cm-2015-TMS.json [hunk 1]
```diff
@@ -0,0 +1,457 @@
+{
+    "url": "http://mapproxy.osm.ch:8080/tiles/KTZUERICH2015/EPSG900913/{z}/{x}/{y}.png?origin=nw",
+    "attribution": {
+        "text": "Kanton Zürich OF 2015",
+        "required": false
+    },
+    "type": "tms",
+    "name": "Kanton Zürich 2015  10cm",
+    "country_code": "CH",
+    "best": true,
```
Comment (bhousel) [inline sources/europe/KantonZuerich10cm-2015-TMS.json]: This source seems to overlap with [Stadt Zürich Luftbild 2011](https://github.com/osmlab/editor-layer-index/blob/b3c1e78c6fdfce260e8a208fc8fa639c15f63492/sources/europe/StadtZuerichOrthophoto2011.json#L10), which is also marked `best: true`.  Should the other one be removed?

Comment (simonpoole) [inline sources/europe/KantonZuerich10cm-2015-TMS.json]: Yes, that has been wrong for a longish time and was removed in OLI when I forked.

**Comment by simonpoole** [comment]
@bhousel I've added two further changes that make the situation more consistent, turned off the best flag for the old Zürich imagery and a polygon for the Canton Aargau stuff from OLI.

**Comment by sfkeller** [comment]
Hi. I'd love to see this PR merged. I'm preparing another MapRoulette campaign among others and would profit from this new imagery.

**Comment by bhousel** [comment]
Thanks @simonpoole!

**Pull request merged**